BUG running quickstart for testpilot

this occurred when following the quick start guide for testpilot:

when I was initially running `npm run package` with npm and node up to date, I was getting a series of errors... but I ran through the entire set up, then came back to it and ran `npm run package` again and no errors appeared, so I'm not sure what fixed it.

but I recreated it just by following the quick start again after re-cloning the repository. here is the error:

<img width="736" alt="29295168-2d3596c8-8110-11e7-91f4-65c9eb0adf68" src="https://user-images.githubusercontent.com/14304935/29378472-6f04e2bc-827c-11e7-9d40-ee858e660af5.png">

These are the commands I ran in order:

`git clone https://github.com/mozilla/testpilot.git`
`nvm use 6`
`cd testpilot/addon`
`npm install`
`npm run package`

during npm install i'm getting:
`UNMET PEER DEPENDENCY babel-core@6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc`
`UNMET PEER DEPENDENCY eslint-plugin-jsx-a11y@5.0.3`
`UNMET PEER DEPENDENCY eslint-plugin-react@7.0.1`
then the following warnings:
`npm WARN babel-loader@7.1.1 requires a peer of babel-core@6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc but none was installed.
npm WARN eslint-config-airbnb@10.0.1 requires a peer of eslint-plugin-jsx-a11y@^2.1.0 but none was installed.
npm WARN eslint-config-airbnb@10.0.1 requires a peer of eslint-plugin-import@^1.13.0 but none was installed.
npm WARN eslint-config-airbnb@10.0.1 requires a peer of eslint-plugin-react@^6.0.0 but none was installed.
npm WARN eslint-config-airbnb-base@5.0.3 requires a peer of eslint-plugin-import@^1.13.0 but none was installed`

it runs through and then after running 'npm run package':
```
npm ERR! Darwin 16.6.0
npm ERR! argv "/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/node" "/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/npm" "run" "package"
npm ERR! node v6.11.2
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! testpilot-addon@2.0.3 package: `cross-env NODE_ENV=production webpack`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the testpilot-addon@2.0.3 package script 'cross-env NODE_ENV=production webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the testpilot-addon package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cross-env NODE_ENV=production webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs testpilot-addon
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls testpilot-addon
```

my npm-debug.log:
```
0 info it worked if it ends with ok
1 verbose cli [ '/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/node',
1 verbose cli   '/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/npm',
1 verbose cli   'run',
1 verbose cli   'package' ]
2 info using npm@3.10.10
3 info using node@v6.11.2
4 verbose run-script [ 'prepackage', 'package', 'postpackage' ]
5 info lifecycle testpilot-addon@2.0.3~prepackage: testpilot-addon@2.0.3
6 silly lifecycle testpilot-addon@2.0.3~prepackage: no script for prepackage, continuing
7 info lifecycle testpilot-addon@2.0.3~package: testpilot-addon@2.0.3
8 verbose lifecycle testpilot-addon@2.0.3~package: unsafe-perm in lifecycle true
9 verbose lifecycle testpilot-addon@2.0.3~package: PATH: /Users/charlesbaker/.nvm/versions/node/v6.11.2/lib/node_modules/npm/bin/node-gyp-bin:/Users/charlesbaker/Desktop/Git Contribute/testcase/testpilot/addon/node_modules/.bin:/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin:/Users/charlesbaker/gcloud_sub_dir/google-cloud-sdk/bin:/Users/charlesbaker/npm/lib/node_modules/grunt-cli/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/ImageMagick/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/charlesbaker/.npm-global/bin/:/Users/charlesbaker/Library/Python/2.7/bin/
10 verbose lifecycle testpilot-addon@2.0.3~package: CWD: /Users/charlesbaker/Desktop/Git Contribute/testcase/testpilot/addon
11 silly lifecycle testpilot-addon@2.0.3~package: Args: [ '-c', 'cross-env NODE_ENV=production webpack' ]
12 silly lifecycle testpilot-addon@2.0.3~package: Returned: code: 1  signal: null
13 info lifecycle testpilot-addon@2.0.3~package: Failed to exec package script
14 verbose stack Error: testpilot-addon@2.0.3 package: `cross-env NODE_ENV=production webpack`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (/Users/charlesbaker/.nvm/versions/node/v6.11.2/lib/node_modules/npm/lib/utils/lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (/Users/charlesbaker/.nvm/versions/node/v6.11.2/lib/node_modules/npm/lib/utils/spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:891:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid testpilot-addon@2.0.3
16 verbose cwd /Users/charlesbaker/Desktop/Git Contribute/testcase/testpilot/addon
17 error Darwin 16.6.0
18 error argv "/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/node" "/Users/charlesbaker/.nvm/versions/node/v6.11.2/bin/npm" "run" "package"
19 error node v6.11.2
20 error npm  v3.10.10
21 error code ELIFECYCLE
22 error testpilot-addon@2.0.3 package: `cross-env NODE_ENV=production webpack`
22 error Exit status 1
23 error Failed at the testpilot-addon@2.0.3 package script 'cross-env NODE_ENV=production webpack'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the testpilot-addon package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     cross-env NODE_ENV=production webpack
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs testpilot-addon
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls testpilot-addon
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
```


*after this, if I continue running the rest of quickstart: 
```
cd ..
npm install
echo '127.0.0.1 example.com' | sudo tee -a /etc/hosts
```

and then go back to `/addon` and run the command `npm run package` everything works just fine

--- a/addon/package.json
+++ b/addon/package.json
@@ -23,6 +23,7 @@
   "main": "./bootstrap.js",
   "license": "MPL-2.0",
   "devDependencies": {
+    "babel-core": "6.24.0",
     "babel-plugin-syntax-async-functions": "6.13.0",
     "babel-plugin-syntax-object-rest-spread": "6.13.0",
     "babel-plugin-transform-object-rest-spread": "6.23.0",

--- a/addon/src/lib/channels.js
+++ b/addon/src/lib/channels.js
@@ -41,7 +41,7 @@ export default class WebExtensionChannel {
     this.listeners = new Set();
     this.topic = topic;
     this.targetAddonId = targetAddonId;
-    this.handleEvent = (event: Object) => {
+    this.handleEvent = event => {
       if (event.data) {
         this.notify(event.data, this.targetAddonId);
       }
@@ -69,11 +69,11 @@ export default class WebExtensionChannel {
     this.addonChromeWebNav = null;
   }
 
-  registerListener(callback: Function) {
+  registerListener(callback) {
     this.listeners.add(callback);
   }
 
-  unregisterListener(callback: Function) {
+  unregisterListener(callback) {
     this.listeners.delete(callback);
   }
 

--- a/addon/webpack.config.js
+++ b/addon/webpack.config.js
@@ -31,7 +31,7 @@ const baseConfig = {
         options: {
           // Minimal babel transforms - Firefox supports many ES2015 features
           babelrc: false,
-          presets: ['react'],
+          presets: [],
           plugins: []
         }
       }

--- a/frontend/config.js
+++ b/frontend/config.js
@@ -30,5 +30,10 @@ module.exports = {
 };
 
 // Pull in local debug-config.json overrides
-const tryRequire = require('try-require');
-Object.assign(module.exports, tryRequire('../debug-config.json') || {});
+let debugConfig;
+try {
+  debugConfig = require('../debug-config.json');
+} catch (e) {
+  debugConfig = {};
+}
+Object.assign(module.exports, debugConfig);

--- a/package.json
+++ b/package.json
@@ -100,7 +100,6 @@
     "sass-lint": "1.10.2",
     "sinon": "2.0.0",
     "through2": "2.0.3",
-    "try-require": "1.2.1",
     "vinyl-buffer": "1.0.0",
     "vinyl-source-stream": "1.1.0",
     "yamljs": "0.2.8"

